oneview_redfish_toolkit/tests/blueprints/test_chassis.py

```diff
@@ -143,6 +143,48 @@ def test_get_enclosure_chassis(self, get_map_appliances,
             "{}{}".format("W/", self.enclosure["eTag"]),
             response.headers["ETag"])
 
+    @mock.patch.object(multiple_oneview, 'get_map_resources')
+    @mock.patch.object(multiple_oneview, 'get_map_appliances')
+    def test_get_enclosure_chassis_when_enclosure_do_not_belong_to_rack(
```

Comment:
I'm not an expert on English but this should not be: test_get_enclosure_chassis_when_enclosure_does_not_belong_to_rack?